js/tests/unit/util/sanitizer.spec.js

```diff
@@ -10,17 +10,75 @@ describe('Sanitizer', () => {
       expect(result).toEqual(empty)
     })
 
-    it('should sanitize template by removing tags with XSS', () => {
-      const template = [
-        '<div>',
-        '  <a href="javascript:alert(7)">Click me</a>',
-        '  <span>Some content</span>',
-        '</div>'
-      ].join('')
-
-      const result = sanitizeHtml(template, DefaultAllowlist, null)
+    it('should retain tags with valid URLs', () => {
+      const validUrls = [
+        '',
+        'http://abc',
+        'HTTP://abc',
+        'https://abc',
+        'HTTPS://abc',
+        'ftp://abc',
+        'FTP://abc',
+        'mailto:me@example.com',
+        'MAILTO:me@example.com',
+        'tel:123-123-1234',
+        'TEL:123-123-1234',
```

Comment:
Should we add a line with other telephone number formats, french for instance?

Reply:
This comes from the Angular repo: https://github.com/angular/angular/blob/8ce99f5a0d958d933e4dda8036abc3683b61a9a7/packages/core/test/sanitization/url_sanitizer_spec.ts#L35
IMO we should keep the same list for consistency but mostly because when I'll merge commits from Bootstrap, for this one, I'm pretty sure I'll forget to check that there is a Boosted mode inside this file and it'll be overridden anyway

Reply:
As long as there is no problem when someone use a non-US/Canadian phone number format, it's fine for me.